### PR TITLE
Address reviews regarding the third section of the paper

### PR DESCRIPTION
**Pull request by Hzaatiti**
We might need to discuss some comparison between k-medoid and k-means in the last part as requested by the reviewer ?

### CODE REVIEW

File: papers/geomstats/geomstats.rst [hunk 1]
```diff
@@ -531,38 +531,25 @@ This and several other datasets can be found in the :code:`datasets.data` module
...
-
-    from geomstats.datasets
-        import graph_data_preparation as gdp
-
-    dim = 2
-    max_epochs = 15
-    lr = .05
-    n_negative = 2
-    context_size = 1
-    karate_graph = gdp.Graph(
```
Comment (ninamiolane) [inline papers/geomstats/geomstats.rst:560]: Maybe still put:
`karate_graph = data_utils.load_karate_graph()`
somewhere in the paper?

This would show where the data come from and would be more consistent with the other tutorials, which use `data_utils.load_cities()` and `data_utils.load_connectomes`.

File: papers/geomstats/geomstats.rst [hunk 2]
```diff
@@ -531,38 +531,25 @@ This and several other datasets can be found in the :code:`datasets.data` module
...
-    | max_epochs   | Number of iterations for learning the embedding|
-    +--------------+------------------------------------------------+
-    | lr           | Learning rate                                  |
-    +--------------+------------------------------------------------+
-    | n_negative   | Number of negative samples                     |
-    +--------------+------------------------------------------------+
-    | context_size | Size of the considered context                 |
-    |              | for each node of the graph                     |
-    +--------------+------------------------------------------------+
+    +--------------+-----------------------------------------------------+-------+
```
Comment (ninamiolane) [inline papers/geomstats/geomstats.rst:534]: Great! This was a very good suggestion indeed.

**Review by ninamiolane — CHANGES_REQUESTED** [review]
Great!! Thanks for your reactivity!! 🎉  A small comment about the use of `data_utils.load_karate_graph`.

For K-medoids, we can probably call the Scikit-Learn function on the dissimilarity matrix built using the hyperbolic metric?

https://scikit-learn-extra.readthedocs.io/en/latest/generated/sklearn_extra.cluster.KMedoids.html

**Comment by Hzaatiti** [comment]
> For K-medoids, we can probably call the Scikit-Learn function on the dissimilarity matrix built using the hyperbolic metric?
> 
> https://scikit-learn-extra.readthedocs.io/en/latest/generated/sklearn_extra.cluster.KMedoids.html

Do you mean to answer the reviewer comment or to try out in the example ? 
Looking at the fit function  of Kmedoid in scikit-learn-extra, it computes the dissimilarity matrix  D within it using one of the metrics from the scikit library (or those from scipy.spatial.distance). In its current implementation form it cannot take D as an input. (But I think it would be easy to have a similar class implemented in geomstats for Kmedoids ?)

**Comment by ninamiolane** [comment]
@Hzaatiti answer to the reviewer + add it to the example? And only adding a mention of it in the scipy paper?

Sorry, I thought scikit-learn could take the dissimilarity matrix as input. Yes, a similar class in geomstats would be easy. @YannCabanes has implemented such classes for Riemannian KNN, etc.
